src/playkit.js

```diff
@@ -69,10 +71,14 @@ export {Env};
 // Export base DRM protocol
 export {BaseDrmProtocol};
 
+//Export default html5 engine
+export {Html5}
```

Comment:
why are you exposing this here like this?

Reply:
adding it - so it will be part of the player - and if someone will want to inherit from it and do his own changes